CoreScriptsRoot/Modules/Panel3D.lua

```diff
@@ -0,0 +1,491 @@
+--Panel3D: 3D GUI panels for VR
```

Comment:
I have a feeling you didn't want to add this? I think it was removed at some point